cylc/flow/suite_files.py

```diff
@@ -71,21 +71,23 @@ class KeyInfo():
     """
 
     def __init__(self, key_type, key_owner, full_key_path=None,
-                 suite_srv_dir=None, platform=None, server_held=True):
+                 suite_srv_dir=None, install_target=None, server_held=True):
         self.key_type = key_type
         self.key_owner = key_owner
         self.full_key_path = full_key_path
         self.suite_srv_dir = suite_srv_dir
-        self.platform = platform
+        self.install_target = install_target
         if self.full_key_path is not None:
             self.key_path, self.file_name = os.path.split(self.full_key_path)
         elif self.suite_srv_dir is not None:
             # Build key filename
             file_name = key_owner.value
 
-            # Add optional platform name
-            if key_owner is KeyOwner.CLIENT and self.platform is not None:
-                file_name = file_name + f"_{self.platform}"
+            # Add optional install target name
+            if (key_owner is KeyOwner.CLIENT
+                and key_type is KeyType.PUBLIC
+                    and self.install_target is not None):
+                file_name = file_name + f"_{self.install_target}"
```

Comment:
Maybe use only f-strings and no string concat `file_name = f"{file_name}_{self.install_target}"`

Reply:
Good plan, I'll fix.